File: praktikum 4/tugas no 1 praktikum pemrograman 4/prak01.c
#include <stdio.h>

int main() {
    int kelipatan, jumlah_anak = 50;
    char simbol;

    scanf("%d %c", &kelipatan, &simbol);
   
    for (int i = 1; i <= jumlah_anak; i++) {
        if (i % kelipatan == 0) {
            printf("%c ", simbol);
        } else {
            printf("%d ", i);
        }
    }

    printf("\n");
    return 0;
}

File: praktikum 4/tugas no 1 praktikum pemrograman 4/prak01.py
kelipatan, simbol = input().split()
kelipatan = int(kelipatan)
jumlah_anak = 50

for i in range(1, jumlah_anak + 1):
    if i % kelipatan == 0:
        print(simbol, end=" ")
    else:
        print(i, end=" ")

print()